tool/src/webapp/bbb-autoclose.html

```diff
@@ -26,23 +26,23 @@
 <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
 <script type="text/javascript" language="JavaScript" src="/bbb-tool/lib/jquery-1.3.2.min.js"></script> 
 <script type="text/javascript" language="JavaScript">
-$(document).ready(function(){
-    closeWindow(); closeWindowAlternative();
-});
+
 
 function closeWindow() {
-    var objWindow = window.open(location.href, "_self");
-    objWindow.close(); 
+    var myWindow = window.open('', '_self', '');
```

Comment:
Can you restore the name for the variable. One line less to be changed in this PR.